### PR TITLE
Fix application path in examples.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: src/help-generator_.toit [hunk 1]
```diff
@@ -531,8 +532,21 @@ class HelpGenerator:
     // Reconstruct the full command line, but now with the options next to the
     // commands that defined them.
     full-command := []
+    is-root := true
+    // For examples we don't want the full path that was used to invoke the
```
Comment (kasperl) [inline src/help-generator_.toit:536]:
```suggestion
    // For examples, we don't want the full path that was used to invoke the
```

File: src/help-generator_.toit [hunk 2]
```diff
@@ -531,8 +532,21 @@ class HelpGenerator:
...
+    // command (like `build/bin/artemis`), but only the basename.
+    app-name := invoked-command_
```
Comment (kasperl) [inline src/help-generator_.toit:538]: Should we have a basename of sorts somewhere?

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: src/help-generator_.toit (hunk 1 above)
Comment (floitsch) [reply src/help-generator_.toit]: done.

File: src/help-generator_.toit (hunk 2 above)
Comment (floitsch) [reply src/help-generator_.toit]: added a local `basename_` function.
done.

**Pull request merged**